### ISSUE TITLE
Can "amp-position-observer" used in an inner div scroll

### ISSUE THREAD

**Issue by GhufranShahid** [ampproject/amphtml#30947]
I am using "amp-position-observer" to hid an amp-img from a scrolling div but "amp-position-observer" is only working on the whole page not inner scrolling div.

![image](https://user-images.githubusercontent.com/37533270/97881067-bc0e4c00-1d43-11eb-9d86-91c61486c502.png)

In the image I marked red color on the scroll on which I want to hide the red marked image but it is hiding on blue marked scroller.
 If I am doing it wrong please guide me to the right solution.
You can preview the same functionality on https://www.ppcexpo.com/ppc

**Comment by micajuine-ho** [comment]
Hi @GhufranShahid the `amp-position-observer` component monitors the position of elements relative to the viewport and works in conjunction with other components such as `amp-animation` to trigger other functionality, such as starting an animation. 

What are you trying to accomplish by adding the `amp-position-observer` component to the left-most scroll portion of the page?

**Comment by GhufranShahid** [comment]
Thanks for you reply, @micajuine-ho I want to hide the arrow I marked red(shown in screenshot) on the scroll end of left-most scroll of the page.

**Comment by micajuine-ho** [comment]
Can you provide a test page that shows the unwanted behavior?

**Comment by GhufranShahid** [comment]
You can check the same behavior in https://www.ppcexpo.com/ppc.
When you scroll the left most scroller to the end you will notice that the "down arrow image" will hide, this is the same behavior I want to achieve in AMP.

I want to observe the position of left most scroll using AMP-Position-Observer. How can I achieve this behavior?

**Comment by micajuine-ho** [comment]
Yes this should be possible. Looking at the [documentation here](https://amp.dev/documentation/components/amp-position-observer) for `amp-position-observer`; I would imagine you would have it observe the last element in the list on the side bar. Additionally, you would have to have the blue arrow be created through `amp-animation`, and configure it to animate to invisible (through css property).

Then, configure the position observer to animate the arrow once the last item enters the viewport:

```
<amp-position-observer
  target="last-list-item-id"
  intersection-ratios="1"
  on="scroll:arrowAnimation.seekTo(time=500)" // The end time of your animation
  layout="nodisplay">
</amp-position-observer>
```

**Comment by GhufranShahid** [comment]
![image](https://user-images.githubusercontent.com/37533270/98909845-2eb9bd00-24e4-11eb-81d1-4c7fc85b62ce.png)

Here is my code:

                      <amp-position-observer target="lastli"
                                           intersection-ratios="1"
                                           on="scroll:arrowAnimation.seekTo(time=500)" 
                    layout="nodisplay">
                    </amp-position-observer>
                    <amp-animation id="arrowAnimation" layout="nodisplay">
                        <script type="application/json">
                            {
                            "selector": "#scrollimg",
                            "duration": "1s",
                            "keyframes":  [{"opacity": "0"}]
                            }
                        </script>
                    </amp-animation>
In screenshot I have mentioned 2 scrollers "scroll 1" and "scroll 2", using the code mentioned above, the animation is applying on the image on scroll of "scroll 1" but I need to apply animation on that image on scroll of "scroll 2".

**Comment by micajuine-ho** [comment]
Can you paste all of the code here, so it's easier to reproduce.

**Comment by GhufranShahid** [comment]
Code of the page is attached in file [amptest.zip](https://github.com/ampproject/amphtml/files/5589622/amptest.zip)
